practiceapp/src/main/java/com/practiceapp/practiceapp/controller/user/CreateUserController.java

```diff
@@ -17,20 +19,29 @@
     @Autowired
     private UserService userService;
 
+    @Autowired
+    private ProfileService profileService;
+
     @ApiOperation(value="Save User",notes="User can be verify by verification token",response=String.class)
-    @RequestMapping(path = "/save",method = RequestMethod.POST,consumes = MediaType.APPLICATION_JSON_VALUE)
+    @RequestMapping(path = "/save", method = RequestMethod.POST, consumes = MediaType.APPLICATION_JSON_VALUE)
     public String saveUser(@Valid @RequestBody UserEntity userEntity){
         return userService.saveUser(userEntity);
     }
 
-    @RequestMapping(path = "/{id}",method = RequestMethod.GET,produces = MediaType.APPLICATION_JSON_VALUE)
+    @RequestMapping(path = "/{id}", method = RequestMethod.GET, produces = MediaType.APPLICATION_JSON_VALUE)
     public UserEntity getUser(@PathVariable("id") String id){
         return null;
     }
 
-    @RequestMapping(path = "/",method = RequestMethod.GET,produces = MediaType.APPLICATION_JSON_VALUE)
+    @RequestMapping(path = "/", method = RequestMethod.GET, produces = MediaType.APPLICATION_JSON_VALUE)
     public UserEntity getUserByName(@RequestParam String userName){
        return userService.getByUserName(userName);
     }
 
+    @RequestMapping(path = "/updateProfile", method = RequestMethod.POST, produces = MediaType.APPLICATION_JSON_VALUE)
```

Comment:
It would be good to have the methods related to the profile in a separate file or rename this file.